<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="38908" started-at="2019-10-14T18:20:09Z" finished-at="2019-10-14T18:20:48Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="38908" started-at="2019-10-14T18:20:09Z" finished-at="2019-10-14T18:20:48Z">
      <class name="com.testing.testcases.UITestCases">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.testing.testcases.UITestCases@20322d26]" name="setup" is-config="true" duration-ms="36194" started-at="2019-10-14T18:20:09Z" finished-at="2019-10-14T18:20:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="completeProfileTest()[pri:0, instance:com.testing.testcases.UITestCases@20322d26]" name="completeProfileTest" duration-ms="6" started-at="2019-10-14T18:20:46Z" finished-at="2019-10-14T18:20:46Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.testing.testcases.UITestCases.completeProfileTest(UITestCases.java:46)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- completeProfileTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.testing.testcases.UITestCases@20322d26]" name="tearDown" is-config="true" duration-ms="2675" started-at="2019-10-14T18:20:46Z" finished-at="2019-10-14T18:20:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.testing.testcases.UITestCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
